StaticResourceSources/gsAssets/step/What_is_Nonprofit_Cloud_modified_undraw_going_up_ttm5.svg

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I noticed that the SVG's in this PR have some random characters at the end of the filename ("ttm5", "tqcd"). I don't see that format in any other SVG's for this project. Is there a reason these are in the filename?

Reply:
@force2b I just left the name the files had in the [shared drive folder](https://drive.google.com/drive/folders/1NRRMQ7zb8eYPzH18sljs8TGTv4abm7DJ?usp=sharing). But it can be easily changed to match the format of the other SVGs within this project.